app/flatpak-builtins-add-remote.c

```diff
@@ -361,8 +362,13 @@ flatpak_builtin_add_remote (int argc, char **argv,
     return FALSE;
 
   /* We can't retrieve the extra metadata until the remote has been added locally, since
-     ostree_repo_remote_fetch_summary() works with the repository's name, not its URL. */
-  return flatpak_dir_update_remote_configuration (dir, remote_name, cancellable, error);
+     ostree_repo_remote_fetch_summary() works with the repository's name, not its URL.
+     Don't propagate errors here because the remote should be usable already. We
+     might just be offline. */
+  if (!flatpak_dir_update_remote_configuration (dir, remote_name, cancellable, &local_error))
+    g_printerr (_("Warning: could not update extra metadata for '%s': %s\n"), remote_name, local_error->message);
```

Comment:
Instead of `g_printerr()` you can just use `g_warning()` (and remove `Warning:` from the message itself). Note that `g_warning()` also adds a newline.